docs/articles/concepts/analytics-and-operational.md

```diff
@@ -0,0 +1,5 @@
+# Analytics and Operational tier
```

Comment:
Neither, actually! The title should be changed once we have the body here, bet my thought was to use this section to showcase that there are different moving parts of the stream processing platform and depending on where in the pipeline your service is placed you would have different requirements and expectations. This is so that the reader is have those things in mind when either thinking about snapshot/restore or scaling/performance

I don't want to use title like "Architecture concerns for Streaming applications", but ultimately, it's what I envision would be inside